corporate/tests/test_stripe.py

```diff
@@ -2460,7 +2460,8 @@ def test_payment_method_string(self, *mocks: Mock) -> None:
         # If you pay by invoice, your payment method should be
         # "Billed by invoice", even if you have a card on file
         # user = self.example_user("hamlet")
-        # do_create_stripe_customer(user, stripe_create_token().id)
+        # billable_user = BillableUser(user)
+        # billlable_user.create_stripe_customer()
```

Comment:
This test has been commented out for a while. Not sure if we want to try to get it working again or delete it? Updated the commented out code for more or less the changes I made.

Reply:
It was commented out in e7220fd71f57a49fab04434539806ac32ec3b9e0, aka in a rewrite before the billing system entered production use. I think if the test seems like it has a useful role, one can plan to finish it, but certainly it's fine to delete it if not. If you're not sure, it'd be reasonable to leave it be and make a note to come back to it in a month when the billing codebase and test suite is more fully in your mind.

Reply:
Yeah, the test looks like it would be useful to finish. I'll put it on the list for down the line in the billing refactor work.